examples/undocumented/python/modelselection_grid_search_kernel.py

```diff
@@ -75,8 +75,7 @@ def create_param_tree():
 
 def modelselection_grid_search_kernel (num_subsets, num_vectors, dim_vectors):
 	# init seed for reproducability
-	Math.init_random(1)
-	random.seed(1);
+	random.seed(1)
```

Comment:
what's the purpose of this? :)

Reply:
Math.init_random doesn't exist any more, and I just removed the semicolon :"D

Reply:
yeah but i dont see the point of this at all to have in any context, regardless whether that is Math::init_random or random.seed()